### ISSUE TITLE
ROS2: forward slash required in topic name

### ISSUE THREAD

**Issue by alsora** [ros-visualization/rqt_publisher#5]
Hi,

I tried the rqt publisher plugin for ROS2 crystal and I found this bug:

 - If I create a publisher on the topic `/my_topic`, then it's possible to start publishing messages at the specified frequency by clicking on the empty box in the topic column.

- If I create a publisher on the topic `my_topic`, that empty box is not displayed. However I'm still able to publish messages one by one by right clicking on the publisher and selecting `Publish selected once`.

From what I have seen so far ROS2 allows to use topics name without the forward slash, so also in rqt it should be possible to publish messages through it.

If on the other hand you don't want to allow users to publish messages on topics without the forward slash, you should at least prompt a warning that the specified topic name is not correct.

**Comment by brawner** [comment]
You're right, this is something the plugin should support. I'll work on a PR
